silkworm/rpc/core/evm_debug.cpp

```diff
@@ -378,9 +378,11 @@ Task<void> DebugExecutor::trace_call(json::Stream& stream, const BlockNumOrHash&
     const auto& block = block_with_hash->block;
     const auto block_num = block.header.number;
 
+    auto transaction_index = static_cast<std::int32_t>(block_with_hash->block.transactions.size());
```

Comment:
Use `gsl::narrow<int32_t>` instead of `static_cast<std::int32_t>`

Reply:
ok